Signal/src/ViewControllers/SelectRecipientViewController.m

```diff
@@ -463,11 +465,11 @@ - (void)updateTableContents
 
         UILabel *countryCodeLabel = self.countryCodeLabel;
         [countryRow addSubview:countryCodeLabel];
-        [countryCodeLabel autoPinEdgeToSuperviewEdge:ALEdgeLeft withInset:20.f];
```

Comment:
See above - we're now using layout margins to properly align cell contents.  And it's easier, as you can see - way less constants.  "Cutting with the grain vs. cutting against the grain," so to speak.